### DISCUSSION TITLE
Make a List of Rejected Suggestions

### DISCUSSION THREAD

**Discussion by Samq64** [ScratchAddons/ScratchAddons#4920] [Ideas] [answered]
Some suggestions have been rejected, but since they're closed, people don't find them and make duplicates which are also closed. I suggest putting an official list of rejected suggestions somewhere and adding anything that's been suggested at leat twice and closed there and linking it in the issue template.

**Comment by Weredime** [comment, marked as answer]
See #927

**Reply by Samq64** [reply]
Ok thanks.

**Reply by Samq64** [reply]
Also, I'm posting this here because the discussion is locked: #1 doesn't have a closed issue, so I don't think it's rejected and private messgaing should be added because it's been suggested at least 3 times.

**Reply by Weredime** [reply]
I went ahead and unlocked it for you
